src/api/libcellml/model.h

```diff
@@ -271,6 +272,13 @@ class LIBCELLML_EXPORT Model: public ComponentEntity
      */
     void linkUnits();
 
+    /**
+     * @brief Returns an unordered set of @c Units names which are used within the
+     * MathML in the model.
+     * @return unordered set of names of @c Units
+     */
+    std::unordered_set<std::string> cnUnits();
```

Comment:
As far as I can tell, `cnUnits()` is only used internally, so it has nothing to do in the API.

Reply:
Not sure about this one.  We don't use it currently (but I could add a test if you like ;) ) I thought it might be useful to the user to be able to retrieve units referenced from within the MathML rather than added through the API?  Happy to delete if that's not the case.  What do others think?  @hsorby @nickerso ?

Reply:
Definitely doesn't belong in the API.

Reply:
Done.